ArduPlane/servos.cpp

```diff
@@ -949,6 +962,9 @@ void Plane::servos_output(void)
     // run vtail and elevon mixers
     servo_output_mixers();
 
+    //  set control surface servos to neutral
+    neutral_control_surface_servos();
```

Comment:
```suggestion
    landing_neutral_control_surface_servos();
```
Maybe adding landing makes it a little clearer that were not putting the control surfaces to neutral all the time ;)